app/controllers/RegionController.scala

```diff
@@ -33,8 +33,8 @@ class RegionController @Inject() (implicit val env: Environment[User, SessionAut
         val features: List[JsObject] =
           RegionTable.getNeighborhoodsWithUserCompletionStatus(user.userId, regionIds).map { case (region, userCompleted) =>
             val properties: JsObject = Json.obj(
-              "region_id" -> region.regionId,
```

Comment:
Let's actually not change these. This isn't a public API, and is only used internally right now I believe. And I don't see other updates in this PR to make the rest of our code account for these changes!